server/schema/submissions/createSubmission.js

```diff
@@ -0,0 +1,31 @@
+const schema = {
+  required: ['output'],
+  oneOf: [
+    { required: ['subjective_submission'] },
+    { required: ['mcq_submission'] },
+  ],
+  properties: {
+    subjective_submission: {
+      type: 'string',
+    },
+    mcq_submission: {
+      type: 'integer',
+    },
+    output: {
+      type: 'string',
+    },
+  },
+  errorMessage: {
+    required: {
+      output: 'Output required',
+    },
```

Comment:
Add this line to `required`
```
mcq_submission: ' One of mcq_submission and subjective_submission is required '

Reply:
done

Reply:
The reason for this is that if it goes to to end of `oneOf` array and errors out, that means it hasn't satisfied the `required` conditions